libraries/aws_rds_instance.rb

```diff
@@ -0,0 +1,75 @@
+require '_aws'
+
+class AwsRdsInstance < Inspec.resource(1)
+  name 'aws_rds_instance'
+  desc 'Verifies settings for an rds instance'
+  example "
+    describe aws_rds_instance(db_instance_identifier: 'test-instance-id') do
+      it { should exist }
+    end
+  "
+
+  include AwsResourceMixin
+  attr_reader :db_id
```

Comment:
This property should be named in line with the AWS API - `db_instance_identifier`.